tests/__init__.py

```diff
@@ -713,6 +712,34 @@ def processing_tasks(pipeline, suite2p_paramset, caiman2D_paramset, caiman3D_par
     imaging.ProcessingTask.delete()
 
 
+@pytest.fixture
+def trigger_processing_suite2p_2D(pipeline, suite2p_paramset, scan_info):
+    """ Triggers suite2p pipeline on subject1 data """
+    imaging = pipeline['imaging']
+    scan = pipeline['scan']
+
+    key = (scan.ScanInfo * imaging.ProcessingParamSet\
+        & "subject='subject1'").fetch("KEY")[0]
+
+    newkey = key.copy()
+    newkey['session_datetime'] = newkey["session_datetime"].strftime("%Y%m%dDT%H%M%S")
+    output_dir = '/main/test_data/demo/' + '_'.join(str(newkey[x]) for x in newkey)
+    imaging.ProcessingTask.insert1({**key,
+        'processing_output_dir': output_dir,
```

Comment:
Let's declare the `processing_output_dir` such that it is relative to the `root_data_dir` (i.e. `/main/test_data`).

Reply:
@kabilar Thanks! This is a good point. I'll make the change.

Reply:
@kabilar How about this? `output_dir = get_imaging_root_data_dir() / test_data / demo /`

Reply:
Never mind my suggestion above. I'll just make it `processed_output_dir = /main/test_data/`